src/rgw/rgw_lc.cc

```diff
@@ -19,6 +20,12 @@
 #include "rgw_zone.h"
 #include "rgw_string.h"
 
+// this seems safe to use, at least for now--arguably, we should
+// prefer header-only fmt, in general
+#undef FMT_HEADER_ONLY
+#define FMT_HEADER_ONLY 1
```

Comment:
if header-only is preferable, we should probably apply this consistently in cmake: `target_compile_definitions(rgw_common PUBLIC "-DFMT_HEADER_ONLY=1")`

Reply:
ok